modules/databend/src/main/java/org/testcontainers/containers/DatabendContainerProvider.java

```diff
@@ -0,0 +1,26 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.containers.DatabendContainer;
```

Comment:
point to the new package `org.testcontainers.databaend.DatabendContainer`. Otherwise, it will not compile

Reply:
Thanks! Fixed.